src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -137,11 +152,13 @@ const connectBreadcrumb: BreadcrumbConnector = function connectBreadcrumb(
               hierarchicalFacetName
             );
             if (breadcrumb.length > 0) {
+              sendEvent('click', breadcrumb[0]);
               helper
                 .toggleRefinement(hierarchicalFacetName, breadcrumb[0])
                 .search();
             }
           } else {
+            sendEvent('click', facetValue);
```

Comment:
these aren't needed, a breadcrumb can only remove a refinement

Reply:
Oh you're right. Clicking something on a breadcrumb means "going up in the hierarchy", and it's removing a refinement.
Then we shouldn't send any event at all from breadcrumb, because we're supposed to send events when adding a refinement.
Does it make sense to you?

Reply:
I think so

Reply:
https://github.com/algolia/instantsearch.js/pull/4463/commits/1ad06ddf5998c31a6150f0e69917c59c6fedf574